Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessTextMetadata.java

```diff
@@ -122,12 +122,6 @@ public String extractSimpleValue() {
         return getValue();
     }
 
-    /**
-     * Sets the contents of the text input field of this process metadata.
-     *
-     * @param value
-     *            value to be set
-     */
     public void setValue(String value) {
```

Comment:
Should this method not got a `@Override` annotation like in the other places?

Reply:
But this is the `setValue` method. Class `ProcessTextMetadata` is extending class `ProcessSimpleMetadata` and `ProcessSimpleMetadata` is a abstract class with abstract method `setValue`. In the other implementations of the `setValue` method in class `ProcessBooleanMetadata`, class `ProcessDateMetadata` and `ProcessSelectMetadata` is always the method `setValue` defined with the `@Override` annotation but not in class `ProcessTextMetadata`.